zulipterminal/model.py

```diff
@@ -869,7 +874,7 @@ def _handle_typing_event(self, event: Event) -> None:
                         ' ',
                         ('code', user['full_name']),
                         ' is typing...'
-                    ])
+                    ], TYPING_STARTED_EXPIRY_PERIOD)
```

Comment:
We could amend the existing test to check whether the method was called with the required args.